src/dotnet/Program.cs

```diff
@@ -81,6 +81,7 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient = null
             var lastArg = 0;
             var cliFallbackFolderPathCalculator = new CliFallbackFolderPathCalculator();
             using (INuGetCacheSentinel nugetCacheSentinel = new NuGetCacheSentinel(cliFallbackFolderPathCalculator))
+            using (IFirstTimeUseNoticeSentinel firstTimeUseNoticeSentinel = new FirstTimeUseNoticeSentinel(cliFallbackFolderPathCalculator))
```

Comment:
It will be system wide. We will need a separate one for the telemetry and for the cache.